src/app/shared/models/who-did-this.model.ts

```diff
@@ -0,0 +1,6 @@
+export interface WhoDidThis {
+  id: string;
+  name: string;
+  bio: string;
+  pic: string;
+}
```

Comment:
as we talked today in the meeting seems that we want social links to be added here, so maybe you can add those to the model